docs/server/channel.md

```diff
@@ -4,35 +4,46 @@ title: Channel
+The `Channel`(aka `Chatroom`) relational model is an important concept when it comes to building chatting apps. A `Channel` aggregates users and prepares an environment for chatting.  Many relational models can exist in-between `User` and `Channel`. These in-between relationships make it easier for developers to add complex business logic in their chat applications. Among these many realtionships, we will first look at the `Membership` relationship.
 
-Currently, we've managed to have the above scenario in one relation which is `Membership`. 
+When a user grants permission to other users to access a `Channel`, a `Membership`relationship is created between the `Users`, `Membership`, and `Channel`. The `memberships` table has a `userId` and a `channelId` that links `Users` to their `Channel`.
 
-## Entity Design
+ Another scenario is when the user wants to manage the sound(vibrate, silent, etc) of each channel, we can create `Sound` relation. Like this, we can add more relations in between to handle more complex logic.
 
-<img src="https://user-images.githubusercontent.com/27461460/88914167-2de67d80-d29d-11ea-8230-6762a4cfe1b4.png" width="400"/>
+## Entity Relationship Model Diagram
+
+<img src="https://user-images.githubusercontent.com/27461460/88914167-2de67d80-d29d-11ea-8230-6762a4cfe1b4.png" width="1000"/>
 
 The decision was made in our first proposal in designing [Entity Relation Model](https://en.wikipedia.org/wiki/Entity%E2%80%93relationship_model).
 * Note that the above [UML](https://creately.com/blog/diagrams/uml-diagram-types-examples) diagram is not recent and this is keep 
 changing as we continue implementing cool features.
 
-As you can see in the `image`, there are `userAlert`, `userMode` as well as `type` of membership which determines what type of permissions user has in the `Channel`. We usually call this a [single table inheritance](https://en.wikipedia.org/wiki/Single_Table_Inheritance) which many models exist in a table. Therefore, we are writing this so that the contributors won't get confused.
+As you can see in the diagram above, there are `userAlert`, `userMode` as well as `type` of membership which determines what type of membership permissions a user has in the `Channel` (view only, leader ...etc). We usually call this a [single table inheritance](https://en.wikipedia.org/wiki/Single_Table_Inheritance) where a single table can have multiple models. We are presenting this diagram to clarify the relationship model for the creation of channels for our contributors. Note that the features that each table has will continue to change.
 
-## Creating channel
+## Creating a Channel
 
-Many developers may have tried similar implementations since there are millions of chat apps around the world. This is however not that simple when creating a channel with a unique number of users.
+Below is a description of how to find a channel using the `unique` number of users in each channel.
```

Comment:
I know I asked you this question but I'm still unsure of why the unique number of users are needed to find a channel instead of just using the channelId. Are we limiting the same users from creating  multiple private channels with the same people ?

Reply:
Yes~ We follow `Slack`. You can experiment on it.